packages/g/gd-sprest.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "gd-sprest",
+  "filename": "dist/gd-sprest.min.js",
```

```suggestion
  "filename": "gd-sprest.min.js",
```
Auto-update uses `dist` as the base, so we don't need it here.